python/lsst/meas/algorithms/skyObjects.py

```diff
@@ -138,11 +138,27 @@ def generateSkyObjects(mask, seed, config):
 
 
 class SkyObjectsTask(Task):
-    """Generate a list of Footprints of sky objects.
+    """Generate a list of Footprints of sky sources (regions on the sky that
+    do not otherwise have detections).
+
+    Parameters
+    ----------
+    schema : `lsst.sfw.table.Schema`
+        Schema used to create the output `lsst.afw.table.SourceCatalog`,
```

Comment:
Similarly here, with the `~` prefix.

Reply:
This one makes sense, as it's in a sentence instead of the type field of the docstring.